<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mer::Internal::MerEmulatorModeOptionsWidget</class>
 <widget class="QWidget" name="Mer::Internal::MerEmulatorModeOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>984</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="deviceModelLabel">
         <property name="text">
          <string>Device model:</string>
         </property>
         <property name="buddy">
          <cstring>deviceModelComboBox</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="MerDeviceModelComboBox" name="deviceModelComboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
         <property name="insertPolicy">
          <enum>QComboBox::InsertAlphabetically</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>874</width>
          <height>535</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="MerEmulatorModeDetailsWidget" name="optionsWidget" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="addProfileButton">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Add new device model&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>&amp;Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeProfileButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Remove selected device model&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>&amp;Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MerDeviceModelComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">mer/merdevicemodelcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>MerEmulatorModeDetailsWidget</class>
   <extends>QWidget</extends>
   <header location="global">mer/meremulatormodedetailswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>deviceModelComboBox</tabstop>
  <tabstop>addProfileButton</tabstop>
  <tabstop>removeProfileButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
